extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexusPlacements.js

```diff
@@ -0,0 +1,26 @@
+define('ext.wikia.adEngine.lookup.prebid.adapters.appnexusPlacements', [
+	'ext.wikia.adEngine.utils.adLogicZoneParams',
+	'wikia.instantGlobals',
+	'wikia.log'
+], function (zoneParams, instantGlobals, log) {
+	'use strict';
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Use of JavaScript strict mode may result in unexpected behaviour in some browsers. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AStrictMode)